### ISSUE TITLE
Signing keys for repository maedevu.maemo.org/leste seem to have expired.

### ISSUE THREAD

**Issue by MacarioMoreno77** [maemo-leste/bugtracker#710]
I get a EXPKEY signal when running apt update. This has happen in the last 24h.

**Comment by carlosgonz0** [comment]
Same here. @MerlijnWajer where is the path for manual downloading the Maemo-keyring to install it?

**Comment by MacarioMoreno77** [comment]
Wizzup has uploaded a key which can be manually added to apt to work around this:
`wget -O - https://maedevu.maemo.org/main-repo-key.asc | sudo apt-key add -`

**Comment by carlosgonz0** [comment]
> Wizzup has uploaded a key which can be manually added to apt to work around this: `wget -O - https://maedevu.maemo.org/main-repo-key.asc | sudo apt-key add -` (from #maemo-leste in Libera.chat)

Great thanks.

**Comment by MerlijnWajer** [comment]
Yeah, sorry guys... I am thinking we can maybe ship a newer maemo keyring in the extras repo, so that people can get the upgrade that way. Then all they need to do is upgrade twice.

**Comment by carlosgonz0** [comment]
> Yeah, sorry guys... I am thinking we can maybe ship a newer maemo keyring in the extras repo, so that people can get the upgrade that way. Then all they need to do is upgrade twice.

Please make new build image for N900 to jump signing key expired
